README.md

```diff
@@ -213,51 +231,75 @@ using a combination of `bwa` and `samtools` to output a BAM file instead of a SA
 
-- A filter for the module must be created in [`.github/filters.yml`](.github/filters.yml). Please include any paths specific for that tool or upstream of that tool (For example bowtie build is upstream of bowtie align).
+- A filter for the module must be created in [`.github/filters.yml`](.github/filters.yml). If the test workflow you have created invokes more than one tool please include any paths specific for those tool's too e.g. `bowtie build` is upstream of `bowtie align` and they have both been chained together to test the latter.
 
 #### Running Tests Locally
 
-0. Have either `docker`, `singularity` or `conda` installed
-1. See [pytest-workflow installation](https://pytest-workflow.readthedocs.io/en/stable/#installation) for directions to install
-2. Run
+1. Install [`nextflow`](https://nf-co.re/usage/installation)
 
-``` bash
-PROFILE=docker pytest --tag new_module --symlink --wt 2 --kwdof
-PROFILE=conda pytest --tag new_module --symlink --wt 2 --kwdof
-TMPDIR=~ PROFILE=singularity pytest --tag new_module --symlink --wt 2 --kwdof
-alias nf-test="PROFILE=docker pytest --tag new_module --symlink --wt 2 --kwdof"
-```
+2. Install any of [`Docker`](https://docs.docker.com/engine/installation/), [`Singularity`](https://www.sylabs.io/guides/3.0/user-guide/) or [`Conda`](https://conda.io/miniconda.html)
+
+3. Install [`pytest-workflow`](https://pytest-workflow.readthedocs.io/en/stable/#installation)
+
+4. Start running your own tests!
+
+    - Typical command with Docker:
+
+        ```console
+        cd /path/to/git/clone/of/nf-core/modules/
+        PROFILE=docker pytest --tag tests/software/bowtie/build --symlink --wt 2 --keep-workflow-wd
+        ```
+
+    - Typical command with Singularity:
+
+        ```console
+        cd /path/to/git/clone/of/nf-core/modules/
+        TMPDIR=~ PROFILE=singularity pytest --tag tests/software/bowtie/build --symlink --wt 2 --keep-workflow-wd
+        ```
+
+    - Typical command with Conda:
+
+        ```console
+        cd /path/to/git/clone/of/nf-core/modules/
+        PROFILE=conda pytest --tag tests/software/bowtie/build --symlink --wt 2 --keep-workflow-wd
+        ```
 
 ### Documentation
 
-- A module MUST be documented in the [`meta.yml`](software/fastqc/meta.yml) file. It MUST document `params`, `input` and `output`. `input` and `output` MUST be a nested list.
+- A module MUST be documented in the [`meta.yml`](software/TOOL/SUBTOOL/meta.yml) file. It MUST document `params`, `input` and `output`. `input` and `output` MUST be a nested list.
```

Comment:
`name` is optional? (again asking for my friend the linter)

Reply:
This bit still needs a bit of work once we know what and how the `meta.yml` will look. Hence https://github.com/nf-core/modules/pull/152/commits/c92fe8701a0c32d36754fca3dd5fe88b8c37d523 😆